### PR TITLE
Update dg planned enhancement section

### PR DESCRIPTION
**Pull request by Arixeyeion**

### CODE REVIEW

**Review by YuZizhen — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -412,8 +413,21 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Proposed Enhancement**: 
 
-- Modify the pre-defined set of allowed characters to **capture any character instead**. This allows for flexibility since all special characters will be accepted for the `ADDRESS` field.
-- For addresses containing `/` characters, the user will be required to use the `\` character to escape each `/` character similar to the example in Planned Enhancement 1.
+- Modify the pre-defined set of allowed characters to **capture any character instead**. However, the characters `\` and `/` **must** be escaped with `\` in order for them to be recognized as an input to the field as these characters may hinder some operations of FINDvior.
+- For addresses containing `/` characters, the user **must** use the `\` character to escape each `/` character similar to the example in Planned Enhancement 1.
+- These changes allow for flexibility since all special characters will be accepted for the `ADDRESS` field.
+
+### 3. Allow users to enter more special characters for Remark and Meeting Remark fields
```
Comment (KohGuanZeh) [inline docs/DeveloperGuide.md:420]: I think the heading should be for name?

Comment (Arixeyeion) [reply docs/DeveloperGuide.md:420]: oh yes I missed out on that.

**Review by Javiery3889 — APPROVED** [review]
LGTM

**Review by Dethada — APPROVED** [review]

**Pull request merged**